doc/index.rst

```diff
@@ -15,6 +15,7 @@ Restic Documentation
    075_scripting
    080_examples
    090_participating
+   095_considerations
```

Comment:
I would suggest to use a 04-Number for this considerations as they mainly affect how backup works. And instead of "considerations" a better name is maybe "tuning_backup_parameters"? E.g. `047_tuning_backup_parameters`?

Reply:
Agreed.  I had intended to ask for input on where this would fit best, but it slipped my mind.